src/main/java/bbd/jportal2/JPortal2Arguments.java

```diff
@@ -46,7 +49,7 @@
     private List<String> templatePostProcessors = new ArrayList<>();
 
     @Parameter(names = {"--template-location", "-tl"}, description = "Freemarker template location. Default is <current_working_directory>/jportal2_templates")
-    private List<String> templateLocations = Arrays.asList(Paths.get(System.getProperty("user.dir"), "jportal2_templates").toString());
+    private List<String> templateLocations = Arrays.asList(Paths.get(System.getProperty("user.dir"), "jportal_templates").toString());
```

Comment:
why the change from jportal2_templates to jportal_templates?

Reply:
Think it snuck in. I've reverted it.